Harvest.Web/Views/Shared/_Layout.cshtml

```diff
@@ -66,14 +66,14 @@
                         <a class="nav-link" href="/project">Projects</a>
                     </li>
                     }
-                    else if (await UserService.HasOnlyRole(Role.Codes.PI))
+                    else
                     {
                         <li class="nav-item active">
                             <a class="nav-link" href="/project/mine">My Projects</a>
                         </li>
                     }
                     
-                    @if (await UserService.HasAccess(AccessCodes.SystemAccess))
+                    @if (await UserService.HasAccess(AccessCodes.FieldManagerAccess))
```

Comment:
I thought permissions were going to be set by system admins.

Reply:
That would be us. we don't want to manage that.